modules/deploy/pages/deployment-option/cloud/vpc-byo-gcp.adoc

```diff
@@ -0,0 +1,524 @@
++
+.... Principal: `serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]`
+
+.... Role: `roles/iam.workloadIdentityUser`
+
+.... Principal: `serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]`
+
+.... Role: `roles/iam.workloadIdentityUser`
++
+```unset
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[cert-manager/cert-manager]"
+gcloud iam service-accounts add-iam-policy-binding <gke-service-account-name>@<service-project-id>.iam.gserviceaccount.com \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:<service-project-id>.svc.id.goog[external-dns/external-dns]"
+```
+
+== Grant rpk user permissions
+
+Grant the user running `rpk` the following permissions. This can be done through a Google account, a service account, or any principal identity supported by GCP.
+
+.Expand necessary permissions
+[%collapsible]
+====
+* `compute.disks.create`
+* `compute.disks.setLabels`
+* `compute.instanceGroupManagers.create`
+* `compute.instanceGroupManagers.delete`
+* `compute.instanceGroupManagers.get`
+* `compute.instanceGroups.create`
+* `compute.instanceGroups.delete`
+* `compute.instanceTemplates.create`
+* `compute.instanceTemplates.delete`
+* `compute.instanceTemplates.get`
+* `compute.instanceTemplates.useReadOnly`
+* `compute.instances.create`
+* `compute.instances.setLabels`
+* `compute.instances.setMetadata`
+* `compute.instances.setTags`
+* `compute.networks.get`
+* `compute.subnetworks.get`
+* `compute.subnetworks.use`
+* `compute.zones.list`
+* `iam.roles.get`
+* `iam.serviceAccounts.actAs`
+* `iam.serviceAccounts.get`
+* `resourcemanager.projects.get`
+* `resourcemanager.projects.getIamPolicy`
+* `serviceusage.services.list`
+* `storage.buckets.get`
+* `storage.buckets.getIamPolicy`
+* `storage.objects.create`
+* `storage.objects.delete`
+* `storage.objects.get`
+* `storage.objects.list`
+====
+
+- If running `rpk` from a Google account, the user should first acquire new user credentials to use for https://cloud.google.com/sdk/gcloud/reference/auth/application-default/login[Application Default Credentials^].
+
+- If running `rpk` from a service account, the user should create a https://cloud.google.com/iam/docs/keys-create-delete#creating[service account key^], then https://cloud.google.com/docs/authentication/application-default-credentials#GAC[export GOOGLE_APPLICATION_CREDENTIALS^] and https://cloud.google.com/sdk/gcloud/reference/config/set[set the account as the default in gcloud^]:
++
+```unset
+export GOOGLE_APPLICATION_CREDENTIALS=<keyfile for service account>
+gcloud config set account $SERVICE_ACCOUNT@$PROJECT_ID.iam.gserviceaccount.com
+```
+
+== Troubleshoot
```

Comment:
At this point the user should have set up all of the pre-requisite resources in GCP, now they will need to go into the Redpanda Cloud UI and create a cluster. Should we have screenshots and walkthrough of that phase?

Reply:
We don't generally include screenshots in our doc, since things change too frequently.